src/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```diff
@@ -148,5 +148,8 @@
   <ItemGroup Condition="'$(TargetsOSX)'=='true'">
     <TestCommandLines Include="ulimit -Sn 4096" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
What is this?  (and below)

Reply:
No idea.  VS must have added it automatically and I didn't notice. I'll remove it.